src/deploy.ts

```diff
@@ -43,7 +43,7 @@ namespace Deploy {
                 ports: [
                     {
                         name: `${port}-tcp`,
-                        port,
+                        port: parseInt(port, 10),
```

Comment:
I usually use `Number(port)` to convert string -> int. How does this behave if the user inputs something invalid like "foo" or "0.1" ? do we have handling for that?

Reply:
No, we are not handling that.
I think `Number(port)` is a good way to achieve that.